src/bpftrace.cpp

```diff
@@ -120,29 +128,79 @@ int BPFtrace::add_probe(ast::Probe &p)
     }
     else
     {
-      attach_funcs.push_back(attach_point->func);
+      if (probetype(attach_point->provider) == ProbeType::usdt && !attach_point->ns.empty())
```

Comment:
This is needed in case the probe was fully specified, to ensure that the format is fully specified later (PROVIDER:FNAME)